lib/Map/LeafletVisualizer.js

```diff
@@ -178,22 +180,37 @@ LeafletGeomVisualizer.prototype.update = function(time) {
     throw new DeveloperError(i18next.t("map.leafletVisualizer.devError3"));
   }
   //>>includeEnd('debug');
+  const bounds = this._leafletScene.map.getBounds();
+  let applyLocalisedAntiMeridianFix = false;
+  let currentBoundsType = 0;
+  if (
+    bounds._southWest.lng > 140 &&
+    (bounds._northEast.lng < -140 || bounds._northEast.lng > 180)
+  ) {
+    applyLocalisedAntiMeridianFix = true;
+    currentBoundsType = 1;
+  } else if (
+    (bounds._southWest.lng > 180 || bounds._southWest.lng < -140) &&
+    bounds._northEast.lng < -140
```

Comment:
Also needs a comment. Is this one is detecting looking just to the west of the antimeridian, or possibly at the antimeridian and being zoomed in?